packit_service/worker/helpers/build/copr_build.py

```diff
@@ -452,7 +452,7 @@ def submit_copr_build(self, script: Optional[str] = None) -> Tuple[int, str]:
                     },
                 )
 
-        except CoprRequestException as ex:
+        except (CoprRequestException, CoprAuthException) as ex:
             if "You don't have permissions to build in this copr." in str(
```

Comment:
do we still need to match the text of the exception if it's separate exception now?